### ISSUE TITLE
Newline changes in newest bigshot (Avalon only?)

### ISSUE THREAD

**Issue by mikeviotti** [elanthia-online/lich-5#266]
Since the newest bigshot changes, it seems like newlines are missing:

![image](https://user-images.githubusercontent.com/38704938/201955845-24c4f01e-1842-4a73-9c6e-8b029d5614c6.png)

![image](https://user-images.githubusercontent.com/38704938/201956204-e05f2cbf-850f-451f-a658-0cab8abac43c.png)

I don't know if this is a bigshot issue or a lich issue. I assume it's Avalon-specific.

```
>;version bigshot
--- Lich: version active.
Installed script versions: bigshot==4.13.7
--- Lich: version has exited.
>
>;e echo LICH_VERSION
--- Lich: exec1 active.
[exec1: 5.6.2]
--- Lich: exec1 has exited.
```

**Comment by OSXLich-Doug** [comment]
Which version of Avalon?  I'm seeing this in the beta Avalon, but not in 4.3.5 .

**Comment by mikeviotti** [comment]
The one I downloaded here: https://www.play.net/gs4/play/avalon-info.asp

Simu's website says 4.3.5. In game, when I do "About Avalon," I get `Avalon (4.3.5 beta)`. I definitely haven't done any custom installations.

**Comment by mrhoribu** [comment]
This should be resolved for Wizard/Avalon users via #269 being merged into staging.
